moped-database/migrations/1715277419652_add_moped_proj_partners_audit_triggers/down.sql

```diff
@@ -0,0 +1,21 @@
+-- Drop triggers to update audit fields on moped_proj_partners and parent project on insert or update of moped_proj_partners
+DROP TRIGGER IF EXISTS update_moped_proj_partners_and_project_audit_fields ON moped_proj_partners;
+
+-- Drop fk constraints on user audit fields on moped_proj_partners
+ALTER TABLE moped_proj_partners
+DROP CONSTRAINT project_types_created_by_fkey,
+DROP CONSTRAINT project_types_updated_by_fkey;
```

Comment:
Could you update these to drop the project partners fk constraints?

Reply:
oh my gosh thanks for catching this! 🍝